_overviews/scala3-migration/incompat-dropped-features.md

```diff
@@ -226,6 +226,45 @@ class Fizz private (val name: String) extends Bar {
 {% endtab %}
 {% endtabs %}
 
+Another use case for early initializers in Scala 2 is private state in the subclass that is accessed (through an overridden method) by the constructor of the superclass:
+
+{% tabs scala-2-initializer_5 %}
+{% tab 'Scala 2 Only' %}
+~~~ scala
+class Adder {
+  var sum = 0
+  def add(x: Int): Unit = sum += x
+  add(1)
+}
+class LogAdder extends Adder {
+  private var added: Set[Int] = Set.empty
+  override def add(x: Int): Unit = { added += x; super.add(x) }
+}
```

Comment:
Did you mean to actually show the early init here?
```suggestion
class LogAdder extends {
  private var added: Set[Int] = Set.empty
} with Adder {
  override def add(x: Int): Unit = { added += x; super.add(x) }
}
```

Reply:
Ah, yeah :-)